src/organisms/MainNavBar.jsx

```diff
@@ -15,8 +17,8 @@ function MainNavBar() {
     if (state.rightMenu === '') { setState({ ...state, rightMenu: ('show') }); } else setState({ ...state, rightMenu: ('') });
   }
 
-  function handleLogout() {
-    store.dispatch(LoginChange(false));
+  async function handleLogout() {
+    await firebase.auth().signOut();
     window.location.hash = '/';
```

Comment:
`window.refresh()` I think would also do the trick for a hard refresh

Reply:
If the user is logged in on the "View profile Page", their navigation bar is "#/viewprofilepage".  If I log them out, i depend on firebase and redux to handle the logout, and the window.location.hash just sends them back to the home page so their navigation isn't pointed at a page they can't access anymore.  If I do window.location.reload(), that doesn't change anything, they are still pointed at the '#/viewprofilepage" even if they are logged out and staring at the home page.  Is there another advantage to window.location.reload?

Reply:
Makes sense with what you're doing. Didn't realize some of the complexity behind the scenes.